### DISCUSSION TITLE
REST API- safe parameters

### DISCUSSION THREAD

**Discussion by draew6** [go-jet/jet#294] [Q&A] [answered]
Hello, as I used only ORMs until now i wasnt afraid of SQL Injections but now i found this perfect library and want to build stack around it. My question is, when i receive parameter userID for example in body of request (let's say echo), do i need somehow sanitize it before i put it here instead of number 12?
`.WHERE(
		User.ID.EQ(Int(12)),
)`

or i can just put there variable and it will be safe?

**Comment by houtn11** [comment, marked as answer]
Yeah, it is safe to just put a variable. Jet will create a parametrized query for you.   
You can see a query and parameters with `stmt.Sql()`.

**Reply by draew6** [reply]
Thank you very much
